### DISCUSSION TITLE
Emails are not getting x-send-later header values.

### DISCUSSION THREAD

**Discussion by geg-ck** [Extended-Thunder/send-later#470] [Q&A] [answered]
Issue posted here: https://github.com/Extended-Thunder/send-later/issues/464

Sending an email with Send Later puts it in the Drafts folder and the Send Later column shows the date/time the email is scheduled for. But, looking at the email header x-send-later, there is no date/time. Also, looking at the debug log, SL3U says No x-send-later headers in message. The Send Later indicator shows [IDLE]. Then at the schedule time the column info erases. 

I have done all the usual rebuild folder, etc. Could there be a compatibility issues with TB 10.2.x where it doesn't add the x-send-later header to the email?

**Comment by jikamens** [comment, marked as answer]
#464 has been fixed.
